### ISSUE TITLE
customizing and simplifying the pickit.nip

### ISSUE THREAD

**Issue by MrBeatDown** [kolton/d2bot-with-kolbot#2007]
Im running a very large pickit list right now. Its almost 2k lines. I did the math and im running about 440 lines of just small charms to pick from. I basically combined all LLD with With max End game stat charms into 1 pickit and I want to simplify and shorten the list. I commented out one of the bot pickit scirpts in my character config and added my own personal Large list pickit for the 1 list my bot is using. It is running with no problems/errors/ etc, BUT, If it can be simplified and shortened then that is what I want to do.  I have no problem with keeping a 1/10/10 small charm. 1 max damage/ 10 life/ 10 attack rating
Is there a line that I can write so I can eliminate several hundred lines of charms so that I get it all? I just came back to diablo after a 5 year break and im rebuilding my melee account.

My test line will look something like this

[Name] == SmallCharm && [Quality] == 4 # [MAXDamage] >= 1 && [ToHit] >= 10 && [MaxHP] >= 10 

This came back as OK when I ran the nipchecker. Will this pick up all 3 mod  Md/AR/Life Sc's in the game no matter what stat or level?  Can this be pushed further to include other stats like FRW/FRH/STR/DEX and all ran on the same "Master Line"  or would I need to make each line separate for their respective Stat {life/Fhr/Frw/Str/Dex} I could not figure out how to add the other stats to the same line and do not know if this is possible.

I will experiment with this line and add it to my list and see if I start picking up these charms I will report back any success or failure I have. I will also do this for Lc and Gc. I will eventually have a much smaller master pick list. Thanks!

**Comment by 5noop** [comment]
Yes your line is fine, you can probably find your answers here https://github.com/blizzhackers/pickits/blob/master/NipGuide.md/#nip-guide

**Comment by MrBeatDown** [comment]
> Yes your line is fine, you can probably find your answers here https://github.com/blizzhackers/pickits/blob/master/NipGuide.md/#nip-guide

Hey thanks a lot. So I guess I can make 1 master line to cover all sc/lc/gd  of the melee 3 mod  charms in the game. Line came out like this

[Name] == SmallCharm && [Quality] == magic # [MAXDamage] >= 1 && [ToHit] >= 10 && ([MaxHP] >= 10 || [Fhr] >= 5 || [Frw] >= 3 || [dexterity] >= 1 || [strength] >= 1)

Passed the nipchecker

Ill run this first line for charms now and see if I get anything.

**Comment by mf022** [comment]
> ([Name] == SmallCharm || [Name] == LargeCharm || [Name] == GrandCharm)

you can use either
```javascript 
[type] == charm && ...
```
